src/pages/data-management/index.jsx

```diff
@@ -44,7 +44,7 @@ const DataManagementPage = ({ route }) => {
   const unnamedExperimentName = 'Unnamed Analysis';
 
   const createNewProject = (newProjectName, newProjectDescription) => {
-    const numUnnamedExperiments = !existingExperiments[0] ? 0
+    const numUnnamedExperiments = !existingExperiments ? 0
```

Comment:
Change to ?.[0] to keep it closer to original

Reply:
Changed